### PR TITLE
storage: Less buttons in the content tabs

### PR DESCRIPTION
**Pull request by mvollmer**
They should all be in the menu now, but got left behind.

### CODE REVIEW

File: pkg/storaged/content-views.jsx [hunk 1]
```diff
@@ -263,12 +264,46 @@ function create_tabs(client, target, is_partition) {
...
+
+    function swap_stop() {
+        return block_swap.Stop({});
+    }
+
+    if (block_swap) {
+        if (block_swap.active)
+            add_menu_action(_("Stop"), swap_stop);
+        else
+            add_menu_action(_("Stop"), swap_start);
```
Comment (KKoukiou) [inline pkg/storaged/content-views.jsx]: I think you mean Start here.

Comment (mvollmer) [reply pkg/storaged/content-views.jsx]: Yes!  And it should be `block_swap.Active` (uppercase) as well.

**Comment by KKoukiou** [comment]
Before:
Screen Shot 2020-01-29 at 11 17 55


After:
Screen Shot 2020-01-29 at 11 25 44

**Comment by KKoukiou** [comment]
@garrett currently storage page uses this dropdown icon for extra actions. In overview page we recently started using the [split buttons](https://www.patternfly.org/v4/documentation/react/components/dropdown#split-button-action) for the same use case. Can we please decide on one of the two so that we can have a unified approach for the future?
Danke

**Comment by mvollmer** [comment]
> In overview page we recently started using the [split buttons](https://www.patternfly.org/v4/documentation/react/components/dropdown#split-button-action) for the same use case.

Note that there is not always a single primary action like "Format":

image

**Comment by garrett** [comment]
Yeah, I've commented a few times before that this button is incorrect.

It also shouldn't be a split button unless there's a button that stays the same and they're related items. It's not the case here.

We are supposed to be using a Kebab for this concept for something like this, but I'd suggest that we might even want to rethink the UI.

**Pull request merged**